packages/components/inputs/selectable-search-input/src/selectable-search-input.tsx

```diff
@@ -221,6 +221,12 @@ export type TSelectableSearchInputProps = {
    * eg: inputDataProps={[{ 'prop-1': 'value-1' }, { 'prop-2': 'value-2' }]}
    */
   inputDataProps?: Record<string, string>;
+  /**
+   * Map of components to overwrite the default ones, see what components you can override
+   * <br/>
+   * [Props from React select was used](https://react-select.com/props)
+   */
+  components?: ReactSelectProps['components'];
```

Comment:
I believe `components` is a very generic property name in the context of this component.

What do you think to update it to something more specific like `selectComponents` or `selectCustomComponents`? 🤔

Reply:
updated hereL https://github.com/commercetools/ui-kit/pull/2831/commits/aa3cd25ca24a4ba0a42c335678e0d72d346457a3

Thank you